frontend-shared/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@hypothesis/frontend-shared",
+  "version": "1.0.0",
+  "repository": "hypothesis/client/frontend-shared",
```

Comment:
I did not check that, but in truth the repo is really just the same as the client so I'm wondering if we should use the same value.

https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

Reply:
The npm docs say:

> The URL should be a publicly available (perhaps read-only) url that can be handed directly to a VCS program without any modification. It should not be a url to an html project page that you put in your browser. It's for computers.

On that basis, I agree - we should just use the URL to the client's repository, since `git checkout` will check out the whole repository, not a specific subdirectory.